Makefile

```diff
@@ -1,4 +1,4 @@
-## Copyright 2016 Istio Authors
+## Copyright 2017 Istio Authors
```

Comment:
a copyright doesn't go forward; it can be a range like 2016-present but usually the first date matters